plugins/kubernetes/README.md

```diff
@@ -211,3 +211,9 @@ Use an annotation to configure what will to be replaced:
 metadata.annotations.samson/set_via_env_json-metadata.labels.custom: SOME_ENV_VAR
 ```
 Then configure an ENV var with that same name and a value that is valid JSON.
+
+### Disabeling service selector validation
+
+To debug services or to create Gateways it can be useful to disable selector validations.
```

Comment:
create resources that need to reference a selector that doesn't include team/role, you can disable selector validation with:
...

(and then and example of a gateway)